Makefile

```diff
@@ -615,7 +622,7 @@ realclean: clean java_clean
 	@$(RM) -f *.bin debian/changelog*
 
 ### additionally removes generated files not in the repository tree
-distclean: realclean clean_genRules
+distclean: clean_stack realclean clean_genRules
```

Comment:
Oh, what a lucky man :). Unfortunately because you are changing the build system, you need to suck this bitter pill and make sure, that it works on launchpad as well. PPA == private package archiv. hets PPA is used so that ubuntu users just can do an `apt-get install hets`, etc. to install hets & friends. For this one needs to create a debian source package description and upload this incl. the sources to launchpad. It than automatically builds the binary packages and finally publishes them to the public PPA mirrors. If there is an error in the source description package or in the build itself, you need to do all the stuff again (no shortcuts). For details see: https://help.launchpad.net/Packaging